eosio/state_example.env

```diff
@@ -0,0 +1,9 @@
+bbs_account='1vs5bschueot'
```

Comment:
Doesn't this info come from the js config now?

Reply:
accounts of eos should be in state.env